tg_utils/health_check/checks/elvis/backends.py

```
@@ -0,0 +1,11 @@
+from django.conf import settings
+
+from tg_utils.health_check.base_checks import HTTPBasedHealthCheck
+
+
+class ElvisProxyHealthCheck(HTTPBasedHealthCheck):
+    """Check if it is possible to establish HTTP connection to Elvis proxy"""
+    expected_status_code = 400
+
+    def get_url(self):
+        return settings.ELVIS_HOST
```

would be better to raise error with descriptions if this setting is missing